src/mod/endpoints/mod_sofia/sofia.c

```diff
@@ -6885,8 +6887,17 @@ static void sofia_handle_sip_r_invite(switch_core_session_t *session, int status
 		if (status >= 400 && sip->sip_reason && sip->sip_reason->re_protocol && (!strcasecmp(sip->sip_reason->re_protocol, "Q.850")
 				|| !strcasecmp(sip->sip_reason->re_protocol, "FreeSWITCH")
 				|| !strcasecmp(sip->sip_reason->re_protocol, profile->sdp_username)) && sip->sip_reason->re_cause) {
+			char status_str[5];
+			const char * session_ignore_list = switch_channel_get_variable(channel, "ignore_reason_header_by_sip_code");
+			const char * current_ignore_list = !zstr(session_ignore_list) ? session_ignore_list : profile->ignore_reason_header_by_sip_code;
+			switch_snprintf(status_str, sizeof(status_str), "%d", status);
+			if(!zstr(current_ignore_list) && !!switch_stristr(status_str, current_ignore_list)) {
```

Comment:
Are we expecting just one status code (608) from ignore_reason_header_by_sip_code parameter? If not, I think we should have the parameter setup to ignore multiple status code. So that ignore_reason_header_by_sip_code could have this kind of format of string `608,606` and in turn, might have to modify` switch_stristr` here so that `current_ignore_list` is the haystack and `status_str` is the needle

Reply:
It is setup to ignore multiple sip error code you can pass e.g. 603,607,608 in the variable